### ISSUE TITLE
Salvage team needs a "head" role

### ISSUE THREAD

**Issue by Nails-n-Tape** [space-wizards/space-station-14#16833]
## Description

I've heard this sentiment from a few people now and I'm starting to buy in on it, mainly for two reasons:

*Since there's only a single "salvage team" role, the skill gap between two people of the same role can be vastly different. intern/pseudo-head roles might make it easier to gauge the skill of each salvage player by role.

*Since salvage has no head, it's consistently out-of-line with the rest of the cargo dept, resulting in little communication between the salvage team and the rest of the supply personnel. There's just not a lot of reason for salvage to interact with cargo beyond chucking the ore they find into the processor, and a designated "salvage supervisor" could make sure the team's being kept in line and doing efficient work rather than just doing whatever they want. it'd also be useful for coordinating shuttle construction.

**Comment by DogZeroX** [comment]
Sloth and I were thinking about it at some point, since we looked at SS13 codebases like citadel that had Pathfinders.

**Comment by GoodWheatley** [comment]
Salvage already completely ignores cargo and does their own thing because cargo techs have no way to go out and get the stuff salvage brings in, and creating a head salvager further exacerbates that problem by making salvage feel even more like a independent department. 

IMO the salvage specialist role doesn’t even need to exist if salvage’s job is to fund cargo, just give the cargo techs access to salvage equipment and let them do salvage while the shuttle isn’t running.

**Comment by deltanedas** [comment]
what if the specialist bit was actually special, only 1 salvage specialist the rest are cargo techies coming along to bring loot back to sell
salvage specialist is expected to be robust the rest not so much

**Comment by OttoMaticode** [comment]
> what if the specialist bit was actually special, only 1 salvage specialist the rest are cargo techies coming along to bring loot back to sell salvage specialist is expected to be robust the rest not so much

probably a bad idea since cargo techies can literally be brand-new players

**Comment by forgotmyotheraccount** [comment]
If there's any changes to be made, right now I'd lean towards letting cargo techies come along for the ride. It's a fun and exciting experience for a new player to FTL to a hostile location, even if they're mainly there just to lug shit to the shuttle. Expedition planets are also a grief-free zone for newbies to learn how to hack doors, hande firearms and engage in combat. EVA suits and weapons would still be out of reach of the techies, unless the specialists trust them enough to supply them with it.

I feel like a salv "head" would be redundant and wouldn't stick around for very long before going the way of the brig medic. I can't see it being necessary unless expeditions expand to the extent that it becomes necessary to separate it from Cargo and becomes its own department.

**Comment by Cheackraze** [comment]
I agree and maybe the quartermaster role could be reworked into it in some way. Independently, cargo go as a department requires very little in the way of leadership but with expeditions, the salvage team needs more coordination than just casual debris scrapping does.  Whether the quartermaster is simply renamed or worked into something else or re-organized or refactor the department, it might require a conscious player effort in the role to reshape it's function during rounds in addition to any gameplay changes.

**Comment by forgotmyotheraccount** [comment]
Been thinking about it a bit more. QM's problem is that even if Cargo is low maintanence, he's still a bit too important to FTL off on an expedition and leave the department to get seized by nukies.

Maybe the question is, would a salv head need to leave the station to guide inexperienced players through an expedition? QM could have access to a unique crew monitoring console that previews expedition sites, and allows the QM to relay information over the radio to the expedition team; primarily where hostile fauna is and where the safest entry points are.

I don't know a thing about programming so I don't know how feasible that would be in the robust engine, but it'd make QM a bit more of an interesting experience and stop the salvage team from feeling so disconnected from the rest of Cargo.

**Comment by slarticodefast** [comment]
I'm closing this issue since it's more a game-design discussion rather than something that needs fixing.
